File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/vssdkprogressstatusbar/vb/guids.vb
Imports System

Class GuidList
    Private Sub New()
    End Sub

    Public Const guidVSSDKProgressStatusBarPkgString As String = "4483bd48-7110-4ce2-a3e7-f68b158fb22f"
    Public Const guidVSSDKProgressStatusBarCmdSetString As String = "e9641950-5084-4add-9a30-6359787753df"

    Public Shared ReadOnly guidVSSDKProgressStatusBarCmdSet As New Guid(guidVSSDKProgressStatusBarCmdSetString)
End Class

File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/vssdkprogressstatusbar/cs/guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace Microsoft.VSSDKProgressStatusBar
{
    static class GuidList
    {
        public const string guidVSSDKProgressStatusBarPkgString = "44ee151b-7660-47ce-8535-74ec7ae209e6";
        public const string guidVSSDKProgressStatusBarCmdSetString = "3031d3ec-bc24-412d-a150-f661e9913445";

        public static readonly Guid guidVSSDKProgressStatusBarCmdSet = new Guid(guidVSSDKProgressStatusBarCmdSetString);
    };
}